### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jdj1907**

### CODE REVIEW

**Review by miguelvaladas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+**How do I get started?**
+------
+Install the necessary packages that can be found in SECTION
+
+
+**Testing**
+------
+For performance and unit tests, please see SECTION
+
```
Comment (kwansinnn) [inline README.md:16]: Can add a Help section.
Help
Any advise for common problems or issues, please email us at xxxx@yyy.com

Comment (kwansinnn) [inline README.md:16]: Can add Repository Tree or Directory Tree.
I can share a program on how to generate the directory tree.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+------ 
+
+This project builds a simple client/server network for the University of Liverpool's Software Development in Practice module.
+It includes an application to create a dictionary, serialisation it, and send to a server.
+
+
+**How do I get started?**
+------
+Install the necessary packages that can be found in SECTION
+
```
Comment (kwansinnn) [inline README.md:11]: Can add under this section
-Dependencies: Python version, Python IDE platform
-Installing command code: Download the repository or clone it using git clone ....
-Executing program: 
Open the command prompt inside the folder
xxx.py
yyy.py

**Review by kwansinnn — COMMENTED** [review]
Please see Kwan Sin(Charlotte) comment below.

**Review by mr-mdd — APPROVED** [review]
Approved per our reviews.

**Pull request merged**